src/include/storage/metric/stats_aggregator.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include <fstream>
+#include <map>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "catalog/catalog.h"
+#include "catalog/namespace_handle.h"
+#include "catalog/schema.h"
+#include "common/macros.h"
+#include "storage/metric/abstract_raw_data.h"
+#include "storage/metric/thread_level_stats_collector.h"
+#include "util/transaction_test_util.h"
+
+namespace terrier {
+
+namespace settings {
+class SettingsManager;
+}
```

Comment:
I think you can remove this declaration from here and can combine `namespace terrier::storage::metric`

Reply:
Settings manager is not yet merged, so there is not a def file we can include.